action.yaml

```diff
@@ -107,9 +107,22 @@ runs:
           -g '${{ inputs.git-source }}' \
           ${{ inputs.channel }}
 
+      # only run if cache path doesn't already exist
+    - name: Check if cache path exists
+      id: check-cache
+      if: ${{ inputs.cache == 'true' }}
+      shell: bash
+      run: |
+        if [ -d "${{ steps.flutter-action.outputs.CACHE-PATH }}" ]; then
+          echo "Cache path already exists, skipping cache step."
+          echo "flutter-action-cache-exists=true" >> "$GITHUB_OUTPUT"
+        else
+          echo "flutter-action-cache-exists=false" >> "$GITHUB_OUTPUT"
+        fi
```

Comment:
Instead of setting an env var, let's add a new value to outputs of this action (starting at line 54 in this file).

Then please also update the README so that others can learn about this.